Example/Backpack/View Controllers/SettingsViewController.swift

```diff
@@ -0,0 +1,35 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import UIKit
+import Backpack
+
+class SettingsViewController: UITableViewController {
+    @IBOutlet weak var closeButton: Button!
```

Comment:
Similar here do we really need to use a `Backpack.Button` over a regular system button with the appropraite `tintColor`?

Reply:
Dogfooding man!

Reply:
It doesn't feel appropriate to use a Backpack Button in the navigation bar in that case we should provide an API for configuring bar button items

Reply:
But this is an example app, and it works...